tests/wallets/test_lndrest.py

```diff
@@ -0,0 +1,701 @@
+import json
+
+import pytest
+from httpx import HTTPStatusError
+from pytest_httpserver import HTTPServer
+from werkzeug.wrappers import Response
+
+from lnbits.wallets.corelightningrest import settings
```

Comment:
importing settings from `lnbits.wallets.corelightningrest`?

Reply:
yes, if I use `from lnbits.settings import settings` then the test and the wallet will have two different instances of `settings`